opsdroid/cli/utils.py

```diff
@@ -43,7 +44,10 @@ def edit_config(ctx, path):
 
     """
     file = path or DEFAULT_CONFIG_PATH
-    editor = os.environ.get("EDITOR", "vi")
+    if platform.system() == "Windows":
+        editor = os.environ.get("EDITOR", "notepad.exe")
+    else:
+        editor = os.environ.get("EDITOR", "vi")
```

Comment:
Perhaps like this would reduce some duplication of magic strings (`EDITOR`) and also make the coverage checker happier.

```suggestion
    editor = os.environ.get("EDITOR", "notepad.exe" if platform.system() == "Windows" else "vi")
```

Reply:
Thank you for reviewing and awesome suggestion.